test/everlong/units/Tend.t.sol

```diff
@@ -295,4 +295,41 @@ contract TestTend is EverlongTest {
         // Stop the prank.
         vm.stopPrank();
     }
+
+    /// @dev Tests that `TendConfig.extraData` is fully loaded even when the
+    ///      content is greater than a single storage slot.
+    function test_extraData_large() external {
```

Comment:
You'll probably want to add tests for cases that assert when `extraData` is `""`,  `< bytes32`, bytes and `> bytes32` bytes.

I re-ran the original test with a single char `bytes` input and the output wasn't matching the input. 

Ensure you include assertions on `minOutput`, `minVaultSharePrice`, and `positionClosureLimit` when populated with non-zero values as well.

Reply:
> I re-ran the original test with a single char bytes input and the output wasn't matching the input.

I couldn't reproduce this, can you show the example. (but agree with adding different test cases for different extreme cases of slots occupied.)